src/components/AvatarGroup/AvatarGroup.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import styles from './AvatarGroup.st.css';
+import { Avatar } from '../Avatar';
+
+export interface AvatarGroupItem {
+  name?: string;
+  src?: string;
+}
+
+export enum AvatarGroupSize {
+  large = 'large',
+  medium = 'medium',
+  small = 'small',
+  xSmall = 'xSmall',
+  xxSmall = 'xxSmall',
+}
+
+export interface AvatarGroupProps {
+  /** Array of Avatar items. Optional. Avatar item structure: {name?: 'Username', src?: 'https://link-to-image'}. All fields are optional. */
+  items?: AvatarGroupItem[];
+  /** Max amount of items shown in summary. Optional. Defaults to 5. */
+  maxAmount?: number;
+  /** Avatar's size. Optional. Dafaults to 'medium'. One of 'large' | 'medium' | 'small' | 'xSmall' | 'xxSmall'. */
+  size?: AvatarGroupSize;
+}
+
+interface DefaultProps {
+  items: AvatarGroupItem[];
+  maxAmount: number;
+  size: AvatarGroupSize;
+}
+
+/** The Avatar group is made up of a group of avatars and an optional text link. Content in text link can be customized to the product's intent. */
+export class AvatarGroup extends React.Component<AvatarGroupProps> {
+  static displayName = 'AvatarGroup';
+  static defaultProps: DefaultProps = {
+    items: [],
+    maxAmount: 5,
+    size: AvatarGroupSize.medium,
+  };
+
+  render() {
+    const { items, size, maxAmount, ...rest } = this.props;
+
+    return (
+      <div {...styles('root', { size }, rest)}>
+        {items.slice(0, maxAmount).map(({ name, src }) => (
+          <div className={styles.avatarContainer}>
+            <Avatar
+              {...styles('avatar', { size }, rest)}
```

Comment:
The problem lies here.
Instead of passing the the "spreaded" `styles` object, you should only pass the class name, like this: ```className={styles.avatar}```

Reply:
Thought about this. There are few things: 
1) Avatar does not accept `className` prop;
2) Avatar must have `size`

I believe this is related with https://github.com/wix/wix-ui-tpa/pull/138#issuecomment-512679463

Reply:
So, the `Stylable` automatically accepts a `className` prop (that's why we send the `rest` or `props` to the spreaded `styles` object i.e. ```...styles('root', {}, rest)```.
So this works out of the box.
Regarding the size, it should be passed as a prop then, and not in stylable.

Usually I prefer having a stylable only on the `root`.
It makes things a bit more readable and neat IMO.

Reply:
Issue is here:
```
Type '{ className: string; name: string; src: string; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Avatar> & Pick<Readonly<AvatarProps> & Readonly<{ children?: ReactNode; }>, "children" | "name" | "src"> & Partial<Pick<Readonly<...> & Readonly<...>, "size">> & Partial<...>'.
```

Reply:
I can bypass TS thing by spreading props, or improving props. Thats ok.

And talking about `size` we agreed that we will not have all the sizes in avatar, as designers added different sets to Avatar vs AvatarGroup.

Reply:
Thanks to @mykas I found a way to solve it properly. 

I missed some concepts on how Stylable works :)